107/src/test/java/org/ehcache/jsr107/LoadAtomicsWith107Test.java

```diff
@@ -41,9 +41,9 @@
  * LoadAtomicsWith107Test
  */
 public class LoadAtomicsWith107Test {
-  private CacheLoader<Number, CharSequence> cacheLoader;
+  private CacheLoader<Integer, String> cacheLoader;
```

Comment:
They were at one point but aren't anymore. The reason was that with the default config, java.lang.Number and java.lang.CharSequence did not have a corresponding serializer since only java.io.Serializable was registered.

This has changed with my TCK fix so this change can be reverted.